examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -5,12 +5,15 @@ namespace Examples.AspNetCore.Controllers;
 
 using System.Diagnostics;
 using System.Diagnostics.Metrics;
+using System.Security.Cryptography;
 using Examples.AspNetCore;
 using Microsoft.AspNetCore.Mvc;
 
 [ApiController]
 [Route("[controller]")]
+#pragma warning disable CA1515 // Consider making public types internal
 public class WeatherForecastController : ControllerBase
+#pragma warning restore CA1515 // Consider making public types internal
```

Comment:
Because this is an examples project, we may want to consider keeping these files as simple as possible.
I would be okay with disabling CA1515 in the csproj.

Reply:
Fixed in dca53fb0e41001e08ee8f9e663d232171e72c1a4